csp_billing_adapter_microsoft/plugin.py

```diff
@@ -73,7 +76,7 @@ def meter_billing(
     """
 
     status = {}
-    usage = _create_usage_list(dimensions, timestamp)
+    usage = _create_usage_list(dimensions, timestamp, config)
```

Comment:
you need to fix the corresponding tests

FAIL Required test coverage of 90.0% not reached. Total coverage: 74.84%
=================================================================== short test summary info ===================================================================
FAILED tests/unit/test_plugin.py::test_get_msi_token - TypeError: _get_msi_token() missing 1 required positional argument: 'config'
FAILED tests/unit/test_plugin.py::test_get_msi_token_error - TypeError: _get_msi_token() missing 1 required positional argument: 'config'
FAILED tests/unit/test_plugin.py::test_get_msi_token_invalid - TypeError: _get_msi_token() missing 1 required positional argument: 'config'
================================================================ 3 failed, 14 passed in 0.28s =================================================================

Reply:
Fixed